src/pages/de/guides/server-side-rendering.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Serverseitiges Rendern
```

Comment:
```suggestion
title: Serverseitiges Rendern (SSR)
```

Da Nutzer vermutlich nach SSR suchen werden, sollte dieser Begriff auch in den Seitentitel, damit unser Algolia-Suchfeld den Treffer ganz oben anzeigt.